.github/workflows/ci.yml

```diff
@@ -240,16 +240,10 @@ jobs:
         build:
           [
             linux-x64-gnu,
-            linux-x64-musl,
             linux-arm64-gnu,
-            linux-arm64-musl,
-            linux-ia32-gnu,
             darwin-arm64,
             darwin-x64,
-            win32-x64-msvc,
-            win32-x64-gnu,
-            win32-arm64-msvc,
-            win32-ia32-gnu,
```

Comment:
> **Summary:**
> Removed unsupported platforms

I think we need to fix the build for JS runtime on all the platforms, and not remove the platforms

Reply:
Yeah I think we should enable support for unsupported platforms. Working on it!